src/track.ts

```diff
@@ -15,6 +15,9 @@ let _client: AptabaseClient | undefined;
 export function init(appKey: string, options?: AptabaseOptions) {
   const [ok, msg] = validate(Platform.OS, appKey, options);
   if (!ok) {
+    if (_client) {
+      dispose();
```

Comment:
You can remove this now

Reply:
If I really remove these lines, then the message from the following console warn is not correct in the scenario that the user initializes again and thinks that tracking (as the corresponding message says) would be disabled.

Reply:
That's a problem for the consumer of the sdk. It shouldn't initiate the sdk with an empty api key

Reply:
I removed it, even though I think you should at least adjust the warning.